perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/AttributesManagerBlImpl.java

```diff
@@ -7284,7 +7284,30 @@ protected void initialize() {
 		rights.add(new AttributeRights(-1, Role.VOADMIN, Arrays.asList(ActionType.READ, ActionType.WRITE)));
 		attributes.put(attr, rights);
 
+		//urn:perun:vo:attribute-def:def:blockManualMemberAdding
+		attr = new AttributeDefinition();
+		attr.setDisplayName("Block manual member adding");
+		attr.setFriendlyName("blockManualMemberAdding");
+		attr.setNamespace(AttributesManager.NS_VO_ATTR_DEF);
+		attr.setDescription("Allows to disable manually add members to Vo.");
```

Comment:
How about something more straightforward and explanatory, like: ``Block manual addition of Vo members. Users are expected to go through registration.`` (same for the group case).

Reply:
Done.